### PR TITLE
Re-enable pod pre-flight test

### PR DESCRIPTION
**Pull request by frederoni**
This test will ensure that integration through CocoaPods is in order by integrating MapboxNavigation and MapboxCoreNavigation on CI via CocoaPods on the current branch.

@bsudekum 👀

### CODE REVIEW

File: MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock [hunk 1]
```diff
@@ -1,50 +1,59 @@
 PODS:
-  - AWSCore (2.5.3)
```
Comment (bsudekum) [inline MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock]: Would it make sense to gitignore this file since it is actually never used?

Comment (frederoni) [reply MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock]: Bitrise is actually using this to cache some of the dependencies in order to speed up this test but I just noticed it's pointing to a non-existing path. (`./MapboxNavigationTests/CocoaPodsTest/PodInstall/Pods -> ./MapboxNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock`)

File: MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile [hunk 2]
```diff
@@ -3,6 +3,6 @@ use_frameworks!
 
 target 'PodInstall' do
   # The branch of MapboxNavigation and MapboxNavigation will be replaced by Bitrise to use the current branch.
-  pod 'MapboxNavigation', :git => 'https://github.com/mapbox/mapbox-navigation-ios.git', :branch => 'BRANCH_NAME'
-  pod 'MapboxCoreNavigation', :git => 'https://github.com/mapbox/mapbox-navigation-ios.git', :branch => 'BRANCH_NAME'
+  pod 'MapboxNavigation', :git => 'GIT_REPO_URL', :branch => 'BRANCH_NAME'
+  pod 'MapboxCoreNavigation', :git => 'GIT_REPO_URL', :branch => 'BRANCH_NAME'
```
Comment (bsudekum) [inline MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile]: 💯

**Comment by 1ec5** [comment]
Builds are currently failing on master due to a dependency on the script being added in this PR.

**Pull request merged**